spec/features/staff/users_can_view_an_activity_as_xml_spec.rb

```diff
@@ -1,69 +1,76 @@
-RSpec.feature "Users can download an activity as XML" do
```

Comment:
So do we still need the `fund_managers_can_view_an_activity_as_xml_spec.rb` version?

Reply:
No I just forgot to update this one properly!